inst/tutorials/treatment/treatment.Rmd

```diff
@@ -0,0 +1,203 @@
+parameters <- set_drugs(parameters = parameters, drugs = list(AL_params))
+
+# View the parameters for AL:
+parameters$drug_efficacy
+
+```
+
+The final line of the chunk above shows us how we can view the drug parameters currently loaded the parameters list once they have been appended. When simulating interventions that use antimalarial drugs (e.g. clinical treatment), this can provide a useful check to ensure you have loaded the drug parameters correctly. In the chunk below, view the `drug_rel_c` parameter:
+
+```{r view-drug-in-params, exercise = TRUE}
+
+# Establish a list of malariasimulation parameters:
+parameters <- get_parameters() 
+
+# Load the parameters for artemether lumefantrine into the parameters
+parameters <- set_drugs(parameters = parameters, drugs = list(DHA_PQP_params))
+
+# View the drug_rel_c parameter for DHA-PQP:
+
+
+```
+
+## 3. Setting clinical treatment
+
+Now that we know how to load antimalarial drug parameters we can specify how they will be administered to the population. In malariasimulation, we parameterise clinical treatment using the `set_clinical_treatment()` function (run `?set_clinical_treatment()` to view the help file for the function). `set_clinical_treatment()` requires four arguments:
+
+1.  `parameters`: A list of `malariasimulation` parameters (with parameters for drug(s) loaded using `set_drugs()`)
+2.  `drug`: The indices of the drug which we are parameterising clinical treatment for
+3.  `timesteps`: A vector of timesteps on which we want to update clinical treatment coverage
+4.  `coverages`: A vector of clinical treatment coverages (proportion of the clinical infections that will be treated in each timestep).
+
+Note that we need to call `set_clinical_treatment()` for each drug that we are specifying clinical treatment for, and that the `timesteps` and `coverage` inputs need to be vectors of equal length, such that each update to the coverage has a corresponding timestep on which the update should take place. The following chunk shows how to set up and run a simulation with clinical treatment:
+
+```{r set-clin-treat-example, exercise = TRUE}
+
+# Establish a list of malariasimulation parameters:
+parameters <- get_parameters(overrides = list(human_population = 1000)) 
+
+# Load the parameters for artemether lumefantrine into the parameters
+parameters <- set_drugs(parameters = parameters, drugs = list(AL_params))
+
+# Specify a clinical treatment regime for artemether lumefantrine:
+parameters <- set_clinical_treatment(parameters = parameters, drug = 1, timesteps = c(0, (1 * 365) + 1), coverages = c(0, 0.6))
+
+# Tune the parameter set to an initial EIR:
+parameters <- set_equilibrium(parameters = parameters, init_EIR = 8)
+
+# Run the simulation
+clinical_treatment_simulation <- run_simulation(timesteps = 3 * 365, parameters = parameters)
+
+# Plot the prevalence through time:
+clinical_treatment_simulation |>
+  ggplot(aes(x = timestep, y = n_detect_lm_730_3650/n_730_3650)) +
```

Comment:
```suggestion
  ggplot(aes(x = timestep, y = n_detect_730_3650/n_730_3650)) +
```

This update isn't in the default malsim yet!

Reply:
Weird - I don't know how it'd be working as I'm not in a malsim project? Have corrected though!